### PR TITLE
Naked by default, re-organizes underwear, adds cow print

### PR DESCRIPTION
**Pull request by SonixApache**
- Standard underwear, 1, is now invisible, making everyone naked by default. - #24182 
 - Changes some of the underwear around to make male- and female underwears equal (so getting genderswapped doesn't suddenly change your underwear that much)
 - Fixes the underwear overlay rendering being hardcoded to 15 - now just checks the list lengths like it should.
 - Adds cow-print underwear for both genders, featuring codersprite from the n3x1s may cry series
RAM_RANCH
If an actual spriter wants to take a go at this, *please* be my guest.
If you think I should ditch the cow print, just sound off in the comments section below.

🆑 
 - imageadd: Added cow-print underwear for both genders. Yee haw.
 - tweak: Both genders are now naked by default.
 - experiment: Changed some underwear numbers around - check your prefs.

### CODE REVIEW

**Comment by anunknownentity03** [comment]
obligatory https://youtu.be/MADvxFXWvwE

**Comment by 9percents** [comment]
travis couldn't handle this and died

**Comment by bathosbathtime** [comment]
Beautiful, stunning, brave. I love it.

**Comment by SonixApache** [comment]
not sure why it didn't pass the travis linter thing

**Comment by bathosbathtime** [comment]
I think it doesn't like the comma in front of the global female underwear list, though I don't know what the comma does or does not do in that context.

File: code/__HELPERS/global_lists.dm [hunk 1]
```diff
@@ -23,8 +23,8 @@ var/global/list/mechas_list = list()				//list of all mechs. Used by hostile mob
 
 //Preferences stuff
 	//Underwear
-var/global/list/underwear_m = list("White", "Grey", "Green", "Blue", "Black", "Mankini", "Love-Hearts", "Black2", "Grey2", "Stripey", "Kinky", "Freedom", "Tea", "Communist", "None") //Curse whoever made male/female underwear diffrent colours
-var/global/list/underwear_f = list("Red", "White", "Yellow", "Blue", "Black", "Thong", "Babydoll", "Baby-Blue", "Green", "Pink", "Kinky", "Freedom", "Communist", "Tea", "None")
+var/global/list/underwear_m = list("None", "White Briefs", "Green Briefs", "Blue Briefs", "Black Briefs", "Grey Briefs", "Mankini", "Love-Hearts Boxers", "Black Boxers", "Grey Boxers", "Stripey Boxers", "Kinky", "Freedom Boxers", "Tea Boxers", "Communist Boxers", "Cowprint Boxers") //Curse whoever made male/female underwear different colours
+, var/global/list/underwear_f = list("None", "White", "Green", "Blue", "Black", "Yellow", "Thong", "Baby-Blue", "Babydoll", "Red", "Pink", "Kinky", "Freedom", "Tea", "Communist", "Cowprint")
```
Comment (Kurfursten) [inline code/__HELPERS/global_lists.dm]: get rid of this `, ` at the start

Comment (SonixApache) [reply code/__HELPERS/global_lists.dm]: good catch

**Comment by Theophrastus1** [comment]
yee haw

**Pull request merged**